src/mmw/mmw/settings/base.py

```diff
@@ -130,6 +130,9 @@ def get_env_setting(setting):
 CELERY_CREATE_MISSING_QUEUES = True
 CELERY_CHORD_PROPAGATES = True
 CELERY_CHORD_UNLOCK_MAX_RETRIES = 60
+# the time limit won't affect the polling tasks since they are short-running
+# the value of 42 coincides with the implied time limit on the polling tasks
+CELERYD_TASK_TIME_LIMIT = 42
```

Comment:
Q: is the implied time limit actually specified somewhere on the client side? Or is it implied elsewhere?